### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by YevhenOstrovskiy**
[DEMO LINK](https://YevhenOstrovskiy.github.io/react_autocomplete/)

### CODE REVIEW

File: src/Input/Input.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    return selectedPerson.name;
+  };
+
+  return (
+    <div className="dropdown-trigger  control has-icons-right">
+      <input
+        type="text"
+        placeholder="Enter a part of the name"
+        className="input"
+        value={handleSelectedPerson()}
```
Comment (rtkachuk-moon) [inline src/Input/Input.tsx:40]: Check your demo. Because sorting is not working) 
Also try not to use ( ) in handleSelectedPerson( ).

**Review by rtkachuk-moon — CHANGES_REQUESTED** [review]

**Review by rtkachuk-moon — APPROVED** [review]
Nice)

File: src/Input/Input.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  setShowPeople = () => { },
+  setAppliedQuery = () => { },
+  setSelectedPerson = () => { },
+  query,
+  setQuery = () => {},
+}) => {
+  const handleSelectedPerson = () => {
+    if (selectedPerson === null) {
+      return query;
+    }
```
Comment (olya-shyian) [inline src/Input/Input.tsx:29]: you can write 
```
if (!selectedPerson) {
  return query;
}
```

File: src/Input/Input.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    <div className="dropdown-trigger  control has-icons-right">
+      <input
+        type="text"
+        placeholder="Enter a part of the name"
+        className="input"
+        value={handleSelectedPerson()}
+        onClick={() => setShowPeople(showPeople)}
+        onChange={handleQueryChange}
+      />
+      {(appliedQuery.length > 0 || selectedPerson) && (
```
Comment (olya-shyian) [inline src/Input/Input.tsx:44]: please don't forget to add a line between inner elements 
check all your files

File: src/Input/Input.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        <span className="icon is-small is-right">
+          <button
+            type="button"
+            className="delete is-small"
+            onClick={() => {
+              setAppliedQuery('');
+              setQuery('');
+              setSelectedPerson(null);
+            }}
```
Comment (olya-shyian) [inline src/Input/Input.tsx:53]: please create a handler for this reset function

File: src/PersonInfo/PersonInfo.tsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+type Props = {
+  person: Person,
+};
+
+export const PersonInfo: React.FC<Props> = ({
+  person,
+
+}) => {
+  return (
+    <div className="dropdown-item">
```
Comment (olya-shyian) [inline src/PersonInfo/PersonInfo.tsx:13]: if you dont have any logic in your component and just have the jsx, then you can return it without word `return`

```
}) => (
  <div className="dropdown-item">
```

check all your components

File: src/PersonInfo/PersonInfo.tsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import classNames from 'classnames';
+import { Person } from '../types/Person';
+
+type Props = {
+  person: Person,
+};
+
+export const PersonInfo: React.FC<Props> = ({
+  person,
+
```
Comment (olya-shyian) [inline src/PersonInfo/PersonInfo.tsx:10]: delete this empty line please

File: src/hooks/useClickOutside.ts [hunk 7]
```diff
@@ -0,0 +1,25 @@
+import React, { useCallback, useEffect } from 'react';
+
+const useClickOutside = (ref: React.MutableRefObject<HTMLElement
+| null>, handler: (arg0: any) => void) => {
```
Comment (olya-shyian) [inline src/hooks/useClickOutside.ts:4]: please do the right formatting

**Review by olya-shyian — CHANGES_REQUESTED** [review]
You did great gob!🙂

**Review by olya-shyian — APPROVED** [review]
Superb!👍

**Pull request closed without merging**